tests/integration/rolling_update/test_rolling_update.py

```diff
@@ -80,6 +81,32 @@ def test_vector_indexer_thread(config):
             flow.search(Document(text='documents after rolling update'))
 
 
+def test_workspace(config, tmpdir):
+    with Flow().add(
+        name='pod1',
+        uses='tests/integration/rolling_update/yaml/simple_index_vector.yml',
```

Comment:
use cur_dir pattern to access these yaml files?

Reply:
thanks, done